WIP-CHANGELOG.md

```diff
@@ -10,6 +13,9 @@
 - Added All instances option to OpenSearch dashboard
 - Add networkpolicy generator chart with common rules
 - Added option to enable harbor GC via config
+- Updated harbor chart to `v1.12.1` which upgraded harbor to `v2.8.1`
+  - Dropped support for chartmuseum
+  - Default login page is now OIDC, for local login refer to path `<url>/account/sign-in`
```

Comment:
Nit: shouldn't this be under `changed` or `updated` instead of `added`?

Reply:
Fixed